gap/isomorph.gi

```diff
@@ -554,3 +554,10 @@ function(n, partition, method)
                 "<partition[i]> is the colour of vertex i; in the second\n",
                 "form, <partition[i]> is the list of vertices with colour i,");
 end);
+
+InstallMethod(IsDigraphAutomorphism,
+"for a digraph and a permutation",
+[IsDigraph, IsPerm],
+function(gr, x)
+  return OnDigraph(gr, x) = gr;
```

Comment:
Please first add a check for whether `gr` is a multidigraph, if so, then have an error message saying it doesn't work for digraphs with multiple edges.